lib/fhir/resources/location.js

```diff
@@ -8,21 +8,24 @@
 'use strict'
 
 const FhirCommon = require('../common')
-const constants = require('../../constants')
 const QueryUtils = require('../query-utils')
 const QueryValidator = require('../query-validator')
+const QueryBuilder = require('../query-builder')
 
 module.exports = (mongo) => {
   const fhirCommon = FhirCommon(mongo)
   const queryUtils = QueryUtils(mongo)
   const queryValidator = QueryValidator()
+  const queryBuilder = QueryBuilder(mongo)
 
   let afterReadHook = (ctx, data, callback) => {
     if (ctx.query && ctx.query._summary) {
       fhirCommon.addSubsettedSecurityLabel(data)
       delete data.telecom
       delete data.address
       delete data.position
+      delete data.partOf
+      delete data.endpoint
```

Comment:
This property seems to be STU3 specific.

Reply:
It does, but it does not harm deleting a property that doesn't exist

Reply:
ok cool im fine with keeping it that way. As a sidenote, it would be awesome if we could do this programatically based on the supplied resource definitions. Not sure if this currently comes though in the definitions though

Reply:
I absolutely agree, but didn't want to over extend. I've added a ticket for it to the backlog.

Reply:
ok awesome, thanks for creating the ticket, also think its better to tackle in a separate ticket